webcomponents/tool/src/main/frontend/js/rubrics/sakai-rubrics-list.js

```diff
@@ -127,6 +127,41 @@ export class SakaiRubricsList extends RubricsElement {
       return await this.querySelector(`#rubric_item_${this.rubrics[this.rubrics.length - 1].id} sakai-rubric`).updateComplete;
     })();
   }
+
+  sortRubrics(sortInputValue, ascending) {
+    switch (sortInputValue) {
+      case 'site_name':
+        if (ascending) {
+          this.rubrics = [...this.rubrics].sort((a,b) => a.title.localeCompare(b.title));
```

Comment:
Have you tried just sorting the original array then calling this.requestUpdate()? That way you don't bother making a copy with the spread operator. It will improve performance if you don't have to create that copy to sort on. The reason you often see the spread operator being used on an array in lit-element code is just to trigger the update cycle.